arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Ior.kt

```diff
@@ -185,13 +207,17 @@ public sealed class Ior<out A, out B> {
     return fold({ f(c, it) }, { g(c, it) }, { a, b -> g(f(c, a), b) })
   }
 
+  @Deprecated(
+    NicheAPI + "Prefer when or fold instead",
+    ReplaceWith("  MN.run { fold({ f(it) },{ g(it) },{ a,b -> f(a).combine(g(b)) })}")
```

Comment:
Reason I made this change, besides adding `this.` is that I think `MN.run { }` on the inside is less error-prone. Since `MN.run { }` seems more invasive on surrounding code. No?

Reply:
Yeah. It's actually not needed to run all inside the code. I'm looking into this replacement, because it is not replacing the code correctly.